code/modules/surgery/bones.dm

```diff
@@ -56,26 +56,26 @@
 
 	begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
 		var/obj/item/organ/external/affected = target.get_organ(target_zone)
-		user.visible_message("[user] is beginning to set the bone in [target]'s [affected.name] in place with \the [tool]." , \
-			"You are beginning to set the bone in [target]'s [affected.name] in place with \the [tool].")
+		user.visible_message("[user] is beginning to set the bone in [target]'s [affected.name] in place with \the [tool].</font>" , \
+			"You are beginning to set the bone in [target]'s [affected.name] in place with \the [tool].</font>")
```

Comment:
There's a close </font> tag, but no open <font> tag. I don't think this is supposed to be here at all.

Or maybe it is, and someone forgot to color it accordingly. I don't do surgery.